includes/Services/Contracts/AI_Service_Interface.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * AI Service Interface.
+ *
+ * Defines the contract for AI service implementations.
+ *
+ * @package WordPress\AI\Services\Contracts
+ */
+
+declare( strict_types=1 );
+
+namespace WordPress\AI\Services\Contracts;
+
+use WordPress\AI_Client\Builders\Prompt_Builder_With_WP_Error;
+
+/**
+ * Interface for AI service implementations.
+ *
+ * Provides a consistent API for experimental features to interact with AI providers
+ * without directly coupling to the underlying AI Client implementation.
+ *
+ * @since 0.1.0
+ */
+interface AI_Service_Interface {
```

Comment:
Why an interface? Seems a bit premature at this point I'd say.

* There's only one `AI_Service` implementation.
* The interface mandates having methods like `generate_text()` which I'm not sure is what makes sense for an interface / contract. There could also be an AI service for generating images for example.

For those reasons, I think it's best to remove this interface entirely. There's no real need for it, at least not at this point.